### DISCUSSION TITLE
If I use pubic key just once, is nonce still necessary?

### DISCUSSION THREAD

**Discussion by lg11** [jedisct1/libsodium#1208] [Q&A] [answered]
I generate random public key and use it just for once, and do not use nonce, will be a vulnerability?

**Comment by jedisct1** [comment, marked as answer]
If you encrypt a single message with a key, a static nonce is okay.

But may what you are looking for is [sealed boxes](https://doc.libsodium.org/public-key_cryptography/sealed_boxes) ?
